signatures/dilithium/dilithium-shake/bench/avx2_weak_skylake-short-sk.yaml

```diff
@@ -0,0 +1,9 @@
+platform: Intel Core i7-6600U Skylake (Mhz 2600) | GCC 7.3.0
+comment: 'https://pq-crystals.org/dilithium/data/dilithium-submission-nist-round2.zip'
+timings:
+  unit: cycles
+  measurements:
+    gen: 73064
+    enc|sign: 222035
+    dec|vrfy: 78196
+  comment: median timings (the signing algorithm varies in time)
```

Comment:
This doesn't fit the data model. `short sk` would have to be a separate parameter set.

Reply:
I'd leave it like this for now, since it's actually the same parameter set with time/memory tradeoff.

Reply:
yeah, our data model doesn't really cover this well... let's leave it like this for now and think about it later